### ISSUE TITLE
Harishchandra does not reveal runner's grip

### ISSUE THREAD

**Issue by adamnfish** [mtgred/netrunner#6144]
When tagged, the runner's grip is not revealed to the Harishchandra player. It also doesn't put the cards in the log, which based on previous Pull Request and Issues seems like is the expected behaviour.

It even looks like this was fixed a few days ago, but that fix doesn't seem to be in effect? (Perhaps it hasn't been released yet, in which case no worries!)
https://github.com/mtgred/netrunner/pull/6123

It would be super awesome to make sure this works for worlds, because it's the kind of interaction that's not easy to do manually.

![jinteki-bug](https://user-images.githubusercontent.com/29761/141503470-a207d20b-14a8-4082-b957-0c8b06d296f3.png)

p.s. I'd love to help if I can, but am not familiar with Clojure. If anyone can put some pointers here I will try and take a look!

**Comment by cardboardtech** [comment]
I believe the update with PR #6123 has not gone live yet.

Always happy to happy to have more people help out. Not sure what programming languages you are used to, but I found Clojure pretty easy to pick up. Clojure documentation is pretty good with a lot of helpful examples. I learned by just throwing myself at some bugs and have just been learning Clojure and jnet's code as I go.

As for this issue, the problem wasn't that Harishchandra's implementation was broken, but how we were sending information to the clients. We have been making some changes to `diffs.clj` which is used to send the right information to the corp, runner, and spectators. One of our changes made it so it wasn't sending the Runner's hand information to the Corp even though Harishchandra said it should be.
